### ISSUE TITLE
Redesign ecosystem page with respect to project categories

### ISSUE THREAD

**Issue by Andersson007** [ansible-community/community-website#366]
Currently, on https://docs.ansible.com/ecosystem.html, we have all projects mixed together w/o any logical separation.
It's hard to read the page, navigate, etc.

I suggest restructuring the page by grouping the projects into several categories based on their purpose.
Possible categories can be:
1. Main "Core" projects all other projects are orbiting around. These are the projects used the most.
2. User tooling aiming to server the "Core" projects.
3. All other projects, they mostly play an auxiliary role for the first two categories.

Example:

- **Main projects** (can be called differently):
  - ansible-core
  - Asible package included collections
  - AWX

- **Tooling**:
  - ansible-navigator
  - ansible-builder
  - ansible-runner
  - Ansible VS Code Extension
  - ansible-lint
  - Molecule

  
- **Other (auxiliary) projects**:
  - Galaxy NG
  - Ansible Language Server
  - AWX Operator
  - Ansible Rulebook
  - ansible-pylibssh
  - Ansible SDK
  - Ansible Sign
  - Ansible Compat
  - Ansible Development Environment
  - Ansible Pytest
  - Ansible Creator
  - Tox Ansible

- Maybe we could create a separate section called **Dev Tools** and move related projects from the previous section.

Thoughts?

**Comment by Andersson007** [comment]
@samccann i see you're not subscribed to the repo, so pinging you manually

**Comment by samccann** [comment]
Right now the page is alphabetical, which is handy, but I agree, it doesn't help anyone learn/understand how pieces fit together. I like the idea of some groupings, but it will need a lot of review. I for example, would consider galaxy a 'core/main' project, all of devTools could be under 'tooling' or 'content/developer tooling' etc.

Then we have side things like AWX operator (which I'm guessing works with AWX, so should it be 'near' that project or under 'extras/auxilliary'? And similarly the Ansible Language Server afaik is used by the vscode extension..so should it be paired with that?

**Comment by Andersson007** [comment]
> Right now the page is alphabetical, which is handy, but I agree, it doesn't help anyone learn/understand how pieces fit together. I like the idea of some groupings, but it will need a lot of review. I for example, would consider galaxy a 'core/main' project, all of devTools could be under 'tooling' or 'content/developer tooling' etc.
> 
> Then we have side things like AWX operator (which I'm guessing works with AWX, so should it be 'near' that project or under 'extras/auxilliary'? And similarly the Ansible Language Server afaik is used by the vscode extension..so should it be paired with that?

@samccann good questions. Yes, it should be thoroughly discussed on the forum and what is in the description now is to provide some visual representation of a possible solution.
If we are good with the grouping, we could create a forum topic asking people for feedback (giving them an example of how it could be organized and pinging people from involved projects):
- Question 1: sections names + short description if needed (i.e. grouping criteria)
- Question 2: content for each section

The final decision will be made by doc team though considering the collected feedback.
Does it sound OK?

**Comment by Andersson007** [comment]
@samccann ^

**Comment by samccann** [comment]
Yeah worth just moving the conversation to the forum to get other opinions. I suggest you create it as a wiki and then we can edit/update based on feedback comments.

Also, there is a combo-package for https://pypi.org/project/ansible-dev-tools/ that groups a bunch of them already. We could use that list as the 'content creator tools' section

**Comment by oraNod** [comment]
@wbentley15 Sounds good. I can work on this next week.

**Comment by wbentley15** [comment]
Yes and thank you ***@***.***,WalterOn Feb 8, 2024, at 1:54 AM, Don Naro ***@***.***> wrote:﻿
@wbentley15 Sounds good. I can work on this next week.

**Comment by oraNod** [comment]
@Andersson007 reading through this again I think we should consult the forum. Let's not rush this one because we're likely to find people can be very opinionated about how projects are grouped.

For the initial launch, I suggest we keep this as is because alphabetical order is neutral. I will reorder everything - even collections which is now on top - so we have an unbiased blank slate to work with.

After launch, let's take it to the forum and re-ignite some of the project discussion around the ecosystem pages. My line of thinking is that we look for categories aligned with use cases. At cfgmgmtcamp (wink @cybette ) there was a bit of discussion about community use cases when we took at look at the docsite landing pages so we kind of have that as a prompt. Anyway we could start a thread in the forum and send a bullhorn blast to ask community to tell us their use cases - then we could use that for the ecosystem page. Wish I had thought of this before last week!

**Comment by Andersson007** [comment]
@oraNod SGTM
